__tests__/components/editor/GroupChoiceModal.test.js

```diff
@@ -52,24 +50,28 @@ describe('<GroupChoiceModal />', () => {
       expect(wrapper.find(Modal.Body).find('form').find('select').length).toEqual(1)
     })
 
-    it('has the first select option as "PCC"', () => {
-      expect(wrapper.find(Modal.Body).find('form').find('select').find('option')
-        .first()
-        .childAt(0)
-        .text()).toEqual('PCC')
-      expect(wrapper.find(Modal.Body).find('form').find('select').find('option')
-        .first()
-        .prop('value')).toEqual('pcc')
+    it('has the first select option as "Cornell University"', () => {
+      const firstOpt = wrapper.find(Modal.Body).find('form select option').first()
+      expect(firstOpt.text()).toEqual('Cornell University')
+      expect(firstOpt.prop('value')).toEqual('cornell')
     })
 
-    it('has the last select option as "University of Washington"', () => {
-      expect(wrapper.find(Modal.Body).find('form').find('select').find('option')
-        .last()
-        .childAt(0)
-        .text()).toEqual('University of Washington')
-      expect(wrapper.find(Modal.Body).find('form').find('select').find('option')
-        .last()
-        .prop('value')).toEqual('wau')
+    it('has the last select option as "Yale University"', () => {
+      const lastOpt = wrapper.find(Modal.Body).find('form select option').last()
+      expect(lastOpt.text()).toEqual('Yale University')
+      expect(lastOpt.prop('value')).toEqual('yale')
+    })
+
+    it('displays the groups from the config in alphabetical order, with ld4p omitted', () => {
+      const expectedGroups = Object.entries(Config.groupsInSinopia)
```

Comment:
I'm not a big fan of this test because basically it's just duplicating the code that is being added to `GroupChoiceModal.jsx`.  It's saying "this code" behaves the same as "this code".  Not really worth holding the PR over though.

Reply:
yeah, so i had a little consternation over that.  i think the thing of value that it adds is that it's testing that the component renders the select box as desired.  so it's not _solely_ "does this code implement this particular filter and sort?"  but it's close, and i wasn't wild about that.

i'd thought about instead sorting and filtering through some more naive/different approach.  but each one that i started writing was way more unwieldy than the current approach, which seems like the obvious way to do those two things to a list in JS.  doing otherwise felt a little hokey, and i wasn't sure how people would feel about that.

putting the whole sorted list in explicitly felt too fragile, but maybe that's what i should've done, and just called it a day?

anyway, yeah, i'm inclined to merge this as-is if @ndushay doesn't object, and maybe touch this up as follow on work if you think it's worth it?

Reply:
I'm fine with it as a follow up ...